components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/main/java/org/wso2/carbon/identity/api/server/configs/v1/core/ServerConfigManagementService.java

```diff
@@ -483,7 +484,8 @@ public void resetRemoteServerConfig(String logType) {
         RemoteServerLoggerData remoteServerLoggerData = new RemoteServerLoggerData();
 
         validateLogType(logType);
-        remoteServerLoggerData.setLogType(logType);
+        // Backend logic supports logType in Uppercase.
```

Comment:
```suggestion
        // Backend logic only supports logType in Uppercase.
```
check other place too

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/573/commits/ba40b016fd8a8afa84e4dce39cec5735747e69d2